src/api/app/views/webui/requests_listing/_breadcrumb_items.html.haml

```diff
@@ -3,5 +3,9 @@
     %li.breadcrumb-item.text-word-break-all
       %i.fa.fa-cubes
       = link_to @project, project_show_path(@project)
+  - if @package
+    %li.breadcrumb-item.text-word-break-all
+      %i.fa.fa-archive
+      = link_to @project, package_show_path(@project, @package)
```

Comment:
This should be the first condition because `@project` is available for package and project requests.

Reply:
The idea here is to show both project and package for packages, like it would be shown for breadcrumbs otherwise

Reply:
I see. Okay